runtime/runtime_impl.hpp

```diff
@@ -71,6 +73,7 @@ class runtime_impl : public runtime {
 			std::cerr << "You didn't call stop() before destructing this plugin." << std::endl;
 			abort();
 		}
+		assert(errno == 0);
```

Comment:
Can you please add the "this is not a definitive check" statement from the pull request in a comment here for clarity?